selector/__init__.py

```diff
@@ -479,8 +478,15 @@ def select(self, visits: List[Visit],
                     comp_val, comp_instrument = Selector.has_complementary_mode(obs, visit.site)
                     instruments_in_obs.append(comp_instrument)
                     valid_in_obs.append(comp_val)
+                    
                     vishours_of_obs.append(obs.visibility.hours[inight])
 
+                    # TODO: Calibrations should be consider? If this is not account
+                    # TODO: more visits are selected. 
+                    if visit.calibrations:
+                        for cal in visit.calibrations:
+                            vishours_of_obs.append(cal.visibility.hours[inight])
```

Comment:
I don't think you need the `if` statement here. If there are no `visit.calibrations`, then the loop:
```
for cal in visit.calibrations:
```
just won't do anything.